lib/absinthe/pipeline.ex

```diff
@@ -92,6 +92,7 @@ defmodule Absinthe.Pipeline do
       # Prepare for Execution
       Phase.Document.CascadeInvalid,
       Phase.Document.Flatten,
+      {Phase.Document.Complexity, options},
```

Comment:
Good question. If it's non optional I think a relatively high default should be picked. We also need to make sure to propagate the ability to set this option to things like Absinthe.Plug.

I'm inclined to include it by default, it's relatively easy to omit a phase from a pipeline if desired.